src/core/mapping/base_mapper.cc

```diff
@@ -766,17 +767,58 @@ bool BaseMapper::map_legate_store(const MapperContext ctx,
   LayoutConstraintSet layout_constraints;
   mapping.populate_layout_constraints(layout_constraints);
 
+  // QUESTION: is it safe to assume that colocated stores should have the same
+  // privileges / projection/ reduction???
+
```

Comment:
@magnatelee : the question above is for you

Reply:
But would it be ever possible that collective instances are colocated with not collective ?